src/foam/nanos/approval/ApprovableAwareDAO.js

```diff
@@ -134,9 +134,13 @@ foam.CLASS({
         throw new RuntimeException("The only approver of " + modelName + " is the maker of this request!");
       }
 
-      if ( getIsTrackingRequestSent() ){
-        sendSingleRequest(x, request, request.getInitiatingUser());
-        approverIds.remove(request.getInitiatingUser());
+      if ( getIsTrackingRequestSent() ) {
```

Comment:
question: how is this ever false? It looks like it's never set anywhere .. so why even bother having this check - why not just send it all the time?

Reply:
Property on ApprovableAware 'isTrackingRequestSent', user can configure it there will make more sense once we transform this into a rule.